objects/rct2/ride/rct2.ride.twist1.json

```diff
@@ -24,6 +24,13 @@
                     "yellow",
                     "black"
                 ]
+            ],
+		    [
```

Comment:
This extra bracket should be removed, i don't think the object will load correctly with this present.

Reply:
the way it is now follows the convention of the object. Removing it isn't logical as far as I know.

Reply:
Looks like it is correct, it’s just that the indentation is off.

Reply:
My bad, i guess it is just the indentation.

Reply:
it appears correct in notepad++, which is what I used to edit it. I suppose I could edit the indentation in the browser.